code/modules/mob/living/devour.dm

```diff
@@ -44,6 +46,16 @@
 		src << "\red You can't eat yourself!"
 		return 0
 
+	if (devouring == victim)
```

Comment:
nah, this was a robustness measure, to ensure that you don't somehow get stuck in a broken state of being unable to devour anything else.